hivemind/client/switch_moe.py

```diff
@@ -82,7 +82,10 @@ def forward(self, input: torch.Tensor, *args: torch.Tensor, **kwargs: torch.Tens
             try:
                 self._expert_info = next((expert.info for experts_i in chosen_experts for expert in experts_i))
             except grpc.RpcError as e:
-                logger.warning(f"Failed to get RemoteMixtureOfExperts.output_shape: {e}")
+                logger.warning(f"Failed to get RemoteSwitchMixtureOfExperts.output_shape: {e}")
+            except StopIteration:
+                raise RuntimeError("No responding experts found during beam search. Check that UID prefixes and "
+                                   "the grid size are consistent with running Server instances.")
```

Comment:
Here, we can probably raise a warning and proceed with the forward pass, returning simply the input. I'll leave that for the near future